pyproject.toml

```
@@ -13,6 +13,9 @@ packages = [
 
 [tool.poetry.dependencies]
 python = ">=3.10, <3.13"
+boto3 = "*"
+urllib3 = "<2"
+pandas = "*"
```

I would probably set these as pandas<2 will soon be deprecated
```suggestion
pandas = ">2"
pyarrow = ">10"
```